content/en/releases/download.md

```diff
@@ -70,6 +72,11 @@ those derivations are signed in the same way as the multi-architecture manifest
 
 <!-- TODO: describe how to use the signed images -->
 
+Full list of images that are signed can be found [here](/examples/admin/signed-images/auto-generated-list-of-all-signed-images.txt)
+
+To verify signed container images within a Kubernetes cluster, please refer to
+[the corresponding cluster administration documentation](/docs/tasks/administer-cluster/verify-signed-images).
```

Comment:
I'm not clear on what we're saying the reader can do if they follow that link.

Do we mean one of these things:

> You can configure your Kubernetes cluster to verify the integrity of container images, including the
> integrity of Kubernetes components such as the kube-controller-manager. Read [PAGE TITLE]
> to learn how.

or

> You can manually verify the integrity of container images that are part of Kubernetes itself,
> and then [DO THINGS] to ensure that the cluster runs those verified images rather than any
> other.

? If not, what do we mean?

Reply:
This is a good call out for clarity. Let me rewrite the sentence